valuable/src/structable.rs

```diff
@@ -34,6 +34,13 @@ use core::fmt;
 ///             println!("{}: {:?}", field.name(), value);
 ///         }
 ///     }
+///
+///     fn visit_value(&mut self, value: Value<'_>) {
+///         match value {
+///             Value::Structable(v) => v.visit(self),
+///             _ => {}
```

Comment:
nit: should this maybe have the comment that you put in the other examples?
```suggestion
///             _ => {} // do nothing for other types
```